components/User/AdminUserProfilePage.vue

```diff
@@ -80,6 +80,25 @@
         :label="$t('Website')"
         type="url"
       />
+      <UploadGroup
+        :label="$t('Profile picture')"
+        type="drop"
+        accept=".jpeg, .jpg, .png"
+        :hint-text="$t('Max size: 4Mo. Accepted formats: JPG, JPEG, PNG')"
```

Comment:
Is the max size actually applied?

Reply:
This is not a native thing we can do like the formats, so it's never applied on the front-end.

But we could / should work on a better image upload experience with cropping like https://pqina.nl/filepond/